src/osg/LightModel.cpp

```diff
@@ -53,37 +53,41 @@ LightModel::~LightModel()
 void LightModel::apply(State& state) const
 {
 
-    #ifdef OSG_GLES1_AVAILABLE
-    #define glLightModeli glLightModelx
-    #endif
+#ifdef OSG_GLES1_AVAILABLE
+#define glLightModeli glLightModelx
+#endif
 
-    glLightModelfv(GL_LIGHT_MODEL_AMBIENT,_ambient.ptr());
+    glLightModelfv(GL_LIGHT_MODEL_AMBIENT, _ambient.ptr());
 
-    if (state.get<GLExtensions>()->glVersion>=1.2)
+    if (state.get<GLExtensions>()->glVersion >= 1.2)
     {
-        if (_colorControl==SEPARATE_SPECULAR_COLOR)
+        if (_colorControl == SEPARATE_SPECULAR_COLOR)
         {
-            glLightModeli(GL_LIGHT_MODEL_COLOR_CONTROL,GL_SEPARATE_SPECULAR_COLOR);
+            glLightModeli(GL_LIGHT_MODEL_COLOR_CONTROL, GL_SEPARATE_SPECULAR_COLOR);
         }
         else
         {
-            glLightModeli(GL_LIGHT_MODEL_COLOR_CONTROL,GL_SINGLE_COLOR);
+            glLightModeli(GL_LIGHT_MODEL_COLOR_CONTROL, GL_SINGLE_COLOR);
         }
     }
 
-    #ifndef OSG_GLES1_AVAILABLE
-    glLightModeli(GL_LIGHT_MODEL_LOCAL_VIEWER,_localViewer);
-    #endif
+#ifndef OSG_GLES1_AVAILABLE
+    glLightModeli(GL_LIGHT_MODEL_LOCAL_VIEWER, _localViewer);
+#endif
 
-    glLightModeli(GL_LIGHT_MODEL_TWO_SIDE,_twoSided);
+    glLightModeli(GL_LIGHT_MODEL_TWO_SIDE, _twoSided);
 }
 
 #else
 
 void LightModel::apply(State&) const
 {
-    OSG_NOTICE<<"Warning: LightModel::apply(State&) - not supported."<<std::endl;
+    static int i = 1;
```

Comment:
If it is easy to remove all the other changes that are whitespace only in this file we should do that. Really it is just the static int i = 1 changes here that are our changes.